content/en/docs/Getting started/_index.md

```diff
@@ -113,14 +113,6 @@ After logging in, you can check the status of the cluster with the `kubectl` too
 sudo -i
 ```
 
-{{% alert title="Bug setting the KUBECONFIG automatically" color="warning" %}}
-There's a bug that affects all but openSUSE systems that prevents the KUBECONFIG to be set automatically. The fix has been merged and will be part of the v3.1.2 release. In the meantime just follow the steps below to set the KUBECONFIG manually.
```

Comment:
@mauromorales removing it here, doesn't remove it from the old versions of the docs, right? It would be good for people to still see if if they are visiting old versions of the docs.